content/docs/guides/adopting/from_kubernetes.md

```diff
@@ -0,0 +1,389 @@
+            var frontend = guestbook.GetResource<Service>("frontend");
+            return new Dictionary<string, object?>
+            {
+                { "privateIp", frontend.Apply(fe => fe.Spec.Apply(spec => spec.ClusterIP)) },
+            };
+        });
+    }
+}
+```
+
+```go
+// Pulumi Kubernetes support for Go is coming soon:
+// https://github.com/pulumi/pulumi-kubernetes/issues/70
+```
+
+As we can see here, the `getResource` function lets us retrieve an internal resource by type and name, so that we can interact with its properties. These will be strongly typed based on the resource type. Be careful using this, of course, as it makes your code subject to the internal implementation details of the YAML configuration &mdash; however, it's often necessary to find the information you need, like the auto-assigned IP addresses.
+
+Running `pulumi up` will deploy the resources and then export the resulting frontend service's auto-assigned cluster IP address:
+
+```bash
+Updating (dev):
+     Type                              Name          Status
+ +   pulumi:pulumi:Stack               k8s-yaml-dev  created
+ +   └─ kubernetes:yaml:ConfigFile     guestbook     created
+ +      ├─ kubernetes:core:Service     redis-master  created
+ +      ├─ kubernetes:core:Service     frontend      created
+ +      ├─ kubernetes:apps:Deployment  frontend      created
+ +      ├─ kubernetes:core:Service     redis-slave   created
+ +      ├─ kubernetes:apps:Deployment  redis-master  created
+ +      └─ kubernetes:apps:Deployment  redis-slave   created
+
+Outputs:
+    privateIp: "10.52.254.168"
+
+Resources:
+    + 8 created
+```
+
+### Deploying Multiple Kubernetes YAML Files
+
+The `ConfigGroup` resource type is similar to `ConfigFile`. Instead of a single file, it accepts a `files` parameter that contains a list of file paths, file globs, and/or URLs from which to read the YAML configuration from. By default, names are used as-is, however you can specify a `namePrefix` to rewrite the names. One or more `transformations` callbacks can be supplied to arbitrarily rewrite resource configurations on-the-fly before deploying them.
+
+To deploy the Kubernetes Guestbook Application using a colllection of YAML files, first create a `yaml` directory and download them into it:
+
+```bash
+$ mkdir yaml
+$ pushd yaml
+$ curl -L --remote-name \
+    "https://raw.githubusercontent.com/kubernetes/examples/master/guestbook/{frontend-deployment,frontend-service,redis-master-deployment,redis-master-service,redis-slave-deployment,redis-slave-service}.yaml"
+$ popd
+```
+
+This Pulumi program uses `ConfigGroup` to read these YAML files, provision the resources inside of them, and export the resulting IP addresses:
+
+
+{{< langchoose csharp >}}
+
+```javascript
+let k8s = require("@pulumi/kubernetes");
+
+// Create resources from standard Kubernetes guestbook YAML example.
+let guestbook = new k8s.yaml.ConfigFile("guestbook", {
```

Comment:
These examples are all still using `ConfigGroup`.  Looks like possibly this file is still WIP so will hold off on looking more closely at this one.